packages/terra-date-picker/src/DatePicker.scss

```diff
@@ -1,20 +1,23 @@
 @import '~react-datepicker/dist/react-datepicker.min';
-
+/* stylelint-disable selector-class-pattern, selector-max-compound-selectors, max-nesting-depth */
```

Comment:
The selector-class-pattern stylelint rule has been removed in [this PR](https://github.com/cerner/terra-core/pull/841) we can remove the disable now

Reply:
Removed selector-class-pattern. https://github.com/cerner/terra-core/pull/768/commits/079249a481ca7ff307578b55d0d3dfadeae48124